README.md

```diff
@@ -2147,6 +2149,57 @@ val d: Task[Int] =
 
 > Note: Support for ZIO environments (`R` in `ZIO[R, E, A]`) is currently in development. Once implemented, it will be possible to use ZIO effects with environments directly within Kyo computations.
 
+### Resolvers: GraphQL Server via Caliban
+
+`Resolvers` integrates with the [Caliban](https://github.com/ghostdogpr/caliban) library to help setup GraphQL servers.
+
+The first integration is that you can use Kyo effects inside your Caliban schemas.
+- If your Kyo effects is `(Aborts[Throwable] & ZIOs)` or a subtype of it, a Caliban `Schema` can be derived automatically.
+- If your Kyo effect is something else, a Caliban schema can be derived if it has a `Runner` for that effect as part of ZIO environment.
+
+```scala
+// this works by just importing kyo.*
+case class Query(k: Int < Aborts[Throwable]) derives Schema.SemiAuto
+
+// for other effects, you need to extend `SchemaDerivation[Runner[YourEnv]]`
+type Env = Vars[Int] & Envs[String]
```

Comment:
I would prefer not to overload the term `Env` - potentially confusing with `Envs[A]` or ZIO's `R` (which is often type aliased to `Env`....

Can we use `Effects`? or `Pending`?

Reply:
Changed to `CustomEffects`